src/Macroable.php

```diff
@@ -9,25 +9,25 @@
 
 trait Macroable
 {
-    protected static $macros = [];
+    protected static array $macros = [];
 
     /**
      * Register a custom macro.
```

Comment:
These docblocks can be removed, right?

Reply:
They can, yes - I've erred towards leaving docblocks if they have a description.  
Should I remove these particular docblocks?